docs/HowToGuides/FAQ.md

```diff
@@ -132,6 +132,18 @@ git rebase --continue
 ```
 
 ### How do I clean up my git history?
+To fix that, you can do the following:
+```
+# 1. Delete your local branch.
+git branch -D
```

Comment:
I think it would be better to suggest `git rebase -i HEAD~<num-commits>` which allows you do squash amend etc. commits in your branch instead of deleting it and cherry-picking stuff.